sources/packages/web/src/types/contracts/RestrictionContract.ts

```diff
@@ -68,3 +68,21 @@ export enum RestrictionEntityType {
    */
   Institution = "Institution",
 }
+
+/**
+ * Types of notification for restrictions.
+ */
+export enum RestrictionNotificationType {
+  /**
+   * No effect.
+   */
+  NoEffect = "No effect",
+  /**
+   * Warning.
+   */
+  Warning = "Warning",
+  /**
+   * Error.
+   */
+  Error = "Error",
+}
```

Comment:
The assigned constants here begin with Upper case while in this other file it begins with lower case. Is there any pattern we are adhering to ? 
<img width="689" alt="image" src="https://user-images.githubusercontent.com/72251620/171256604-1ab38f54-0daa-4e99-87d1-1bc5d7c59007.png">

Reply:
We are adhering to a pattern that all the enum property names must start with uppercase.  And the key and value not required to Match case and value in some cases.
https://github.com/bcgov/SIMS/wiki/Development-Standards-and-Decisions